package/WindowsManaged/Program.cs

```diff
@@ -259,6 +283,10 @@ static void Main()
                             StopOn = SvcEvent.InstallUninstall,
                         },
                     },
+                    new (LibXmfPath)
+                    {
+                        TargetFileName = Includes.EXECUTABLE_NAME,
```

Comment:
The constant value is "DevolutionsGateway.exe", so this will not work; since we don't need to reference this elsewhere in the installer you could just put the value directly e.g. ` = "xmf.dll"`